src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityOperatorBase.cs

```diff
@@ -28,7 +28,7 @@ internal abstract class SynthesizedRecordEqualityOperatorBase : SourceUserDefine
 
         protected SynthesizedRecordEqualityOperatorBase(SourceMemberContainerTypeSymbol containingType, string name, int memberOffset, DiagnosticBag diagnostics)
             : base(MethodKind.UserDefinedOperator, name, containingType, containingType.Locations[0], (CSharpSyntaxNode)containingType.SyntaxReferences[0].GetSyntax(),
-                   DeclarationModifiers.Public | DeclarationModifiers.Static, hasBody: true, isExpressionBodied: false, isIterator: false, diagnostics)
+                   DeclarationModifiers.Public | DeclarationModifiers.Static, hasBody: true, isExpressionBodied: false, isIterator: false, isNullableEnabled: false, diagnostics)
```

Comment:
>isNullableEnabled: false [](start = 139, length = 24)

Same question here.

Reply:
This should be the correct value for this method.

---
In reply to: [546876982](https://github.com/dotnet/roslyn/pull/49953#discussion_r546876982) [](ancestors = 546876982)